nservicebus/vscode.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Debugging NServiceBus in Visual Studio Code
+summary: How to configure Visual Studio Code to build and debug multiple NServiceBus endoints simultaneously
+component: Core
+reviewed: 2018-08-01
+---
+
+NServiceBus 7 and above is designed to run on .NET Core, which means instead of using Visual Studio as an IDE, users can use the cross-platform [Visual Studio Code](https://code.visualstudio.com/) application (or "VSCode") to build systems with NServiceBus.
+
+While Visual Studio is designed only to run .NET projects, VSCode is a more general-purpose IDE for any language, so there is a little bit of configuration to know how to build and debug .NET projects. Because NServiceBus projects typically involve running multiple startup projects to test multiple endpoints simultaneously, a little extra configuration is needed.
```

Comment:
> so there is a little bit of configuration to know how to build and debug .NET projects

reads strange. Perhaps I was expecting to see "a some configuration to be made".

Reply:
It's the "a little bit" that bothers me. For documentation article sounds too informal.